src/docs/common_tasks/multiple_jvm_versions.md

```diff
@@ -35,10 +35,11 @@ more aliases than javac. Up to JDK 8, you can use either `1.X` or `X`. For 9 and
 include the `1.`.
  * `args`, allows the platform to specify additional, global to that platform, compile arguments.
 
-For runtime configuration, `target` and `jvm_options` are used. `target` is used to determine the
-version of the JVM to use. `jvm_options` is used by some tasks to allow platforms to specify
-platform specific JVM options. These can make transitions between JVM versions smoother by
-allowing compatibility options to be provided globally.
+For runtime configuration, `target`, `strict` and `jvm_options` are used. `target` is used to
+determine the minimum version of the JVM to use. `strict` makes the max version also use the
```

Comment:
Unless I'm missing something, "max version" is not a user-facing term. How about: "`strict` forces the JVM used to be exactly the `target` version"?